src/app/clients/getExpandedClientDetails.ts

```
@@ -313,15 +291,49 @@ export const formatBreakdownOfChildrenFromFamilyDetails = (
     return childDetails.join(", ");
 };
 
-export const formatBreakdownFromArray = <T>(
+const formatBreakdownFromArray = <T>(
     arr: T[] | null | undefined,
-    getName: (item: T) => string | number | null | undefined
+    getName: (item: T) => string | number | null | undefined,
+    getAdditionalInfo?: (item: T) => string | null | undefined
 ): string => {
-    if (!arr || arr.length === 0) {
+    if (!arr?.length) {
         return "-";
     }
-    const names = arr.map((item) => getName(item)).filter(Boolean);
-    return names.join(", ");
+
+    const items = arr
+        .map((item) => {
+            const name = getName(item)?.toString().trim();
+            if (!name) {
+                return null;
+            }
+
+            const extraInfo = getAdditionalInfo?.(item)?.trim();
+            return extraInfo ? `${name} (${extraInfo})` : name;
+        })
+        .filter(Boolean);
+
+    return items.length ? items.join(", ") : "None";
```

Do you think it would be better to use a more descriptive default label than "None" for empty items array?

---

The function isn't solely used for item array breakdowns, so using a more descriptive default label might be misleading in other contexts.